core/src/main/java/io/questdb/cutlass/pgwire/CleartextPasswordPgWireAuthenticator.java

```diff
@@ -494,6 +499,9 @@ private int writeToSocketAndAdvance(State nextState) {
         return Authenticator.NEEDS_WRITE;
     }
 
+    protected boolean verifyPassword(CharSequence username, long passwordPtr, int passwordLen) {
```

Comment:
nit: might be a good idea to leave comment "kept protected for ent" or something like that.

Reply:
done